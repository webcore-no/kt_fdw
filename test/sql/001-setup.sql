set client_min_messages=ERROR;
drop server if exists kt_server cascade;
set client_min_messages=NOTICE;

create  server kt_server foreign data wrapper kt_fdw;
create foreign table kt_test (key text, value text, flag text) server kt_server;
create user mapping for public server kt_server;

drop server if exists kt_server cascade;

create  server kt_server foreign data wrapper kt_fdw;
create foreign table kt_test (key bytea, value bytea, flag text) server kt_server;
create user mapping for public server kt_server;
