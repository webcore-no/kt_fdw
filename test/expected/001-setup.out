set client_min_messages=ERROR;
drop server if exists kt_server cascade;
set client_min_messages=NOTICE;
create  server kt_server foreign data wrapper kt_fdw;
create foreign table kt_test (key text, value text, flag text) server kt_server;
create user mapping for public server kt_server;
drop server if exists kt_server cascade;
NOTICE:  drop cascades to 2 other objects
DETAIL:  drop cascades to foreign table kt_test
drop cascades to user mapping for public on server kt_server
create  server kt_server foreign data wrapper kt_fdw;
create foreign table kt_test (key bytea, value bytea, flag text) server kt_server;
create user mapping for public server kt_server;
